Matchismo/Matchismo/Model/CardMatchingGame.h

```diff
@@ -11,18 +11,19 @@
 
 @interface CardMatchingGame : NSObject
 
-enum GameMode : int {
+typedef enum {
   two = 2,
   three = 3
-};
+} GameMode;
 
 - (instancetype)initWithCardCount: (NSUInteger)count usingDeck:(Deck *)deck;
 - (void)chooseCardAtIndex: (NSUInteger)index;
 - (Card *)cardAtIndex: (NSUInteger)index;
 - (BOOL)startNewGameWithCardCount: (NSUInteger)count usingDeck: (Deck *)deck;
 
 @property (nonatomic, readonly) NSInteger score;
-@property (nonatomic) enum GameMode mode;
-@property (nonatomic, readonly) NSString *lastCardFlipDescription;
+@property (nonatomic) GameMode mode;
+@property (nonatomic, readonly) NSAttributedString *lastMoveDescription;
+@property (nonatomic) NSMutableAttributedString *moveHistory;
```

Comment:
Exposing mutable stuff as a public property is generally a bad idea.  Is it possible to encapsulate this and hide the mutability?

Reply:
Also, the fact that it's both mutable and settable (i.e. not `readonly`) doesn't quite make sense.